google-cloud-datastore/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>com.google.cloud.datastore</groupId>
       <artifactId>datastore-v1-proto-client</artifactId>
-      <version>1.4.1</version>
+      <version>1.5.1</version>
```

Comment:
should this match datastore-v1-protos (i.e. either both 1.5.0 or both 1.5.1)? I know it did not match before, but it seems like it was wrong originally.

Reply:
This version should just be removed at this point, it is in google-cloud-bom now.

Reply:
Removed.